doc/changes/latest.inc

```diff
@@ -70,3 +70,5 @@ API changes
 - The ``show_toolbar`` argument to :class:`mne.viz.Brain` is being removed by deprecation (:gh:`11049` by `Eric Larson`_)
 - New classes :class:`~mne.time_frequency.Spectrum` and :class:`~mne.time_frequency.EpochsSpectrum`, created via new methods :meth:`Raw.compute_psd()<mne.io.Raw.compute_psd>`, :meth:`Epochs.compute_psd()<mne.Epochs.compute_psd>`, and :meth:`Evoked.compute_psd()<mne.Evoked.compute_psd>` (:gh:`10184` by `Daniel McCloy`_)
 - The PSD functions that operate on Raw/Epochs/Evoked instances (``mne.time_frequency.psd_welch`` and ``mne.time_frequency.psd_multitaper``) are deprecated; for equivalent functionality create :class:`~mne.time_frequency.Spectrum` or :class:`~mne.time_frequency.EpochsSpectrum` objects instead and then run ``spectrum.get_data(return_freqs=True)`` (:gh:`10184` by `Daniel McCloy`_)
+- New class  :class:`mne.preprocessing.EOGRegression` created to allow more flexibility when using regression to reduce EOG artifacts (:gh:`11046` by `Marijn van Vliet`_)
+- New parameters ``exclude`` and ``taa`` added to :func:`mne.preprocessing.regress_artifact` to prevent regression from being applied to certain channels and to apply TAA correction respectively (:gh:`11046` by `Marijn van Vliet`_)
```

Comment:
can you expand the acronym TAA here to make the entry more accessible?

Reply:
I ended up removing the parameter completely. YAGNI.